tools/serverpod_cli/lib/src/analyzer/models/validation/restrictions/default.dart

```diff
@@ -151,4 +153,29 @@ class DefaultValueRestriction extends ValueRestriction {
     }
     return errors;
   }
+
+  List<SourceSpanSeverityException> _stringValidation(
+    dynamic value,
+    SourceSpan? span,
+  ) {
+    var errors = <SourceSpanSeverityException>[];
+
+    if (value is! String) {
+      errors.add(
+        SourceSpanSeverityException(
+          'The "$key" value must be a valid string.',
+          span,
+        ),
+      );
+    }
+    String cleanedValue = value.replaceAll(r'\\', '').replaceAll('\'', '');
```

Comment:
Question/Discussion: What if `'` is used in the string itself? Feels like we should support also that symbol. Also, is there a risk of SQL injections through the default value to the database?

Reply:
This is already updated and removed!